CHANGELOG.md

```diff
@@ -1,5 +1,134 @@
 # Change Log
 
+## v9.0.0-beta - 2020-06-01
+
```

Comment:
We need to add information on how to migrate a Celery based deployment to RQ one. @rauchy can you take this?

What I had in mind:
1. Start an instance with our current `docker-compose.yml` setup (from getredash/setup).
2. Update the image to use v9 (you can find latest in redash/preview docker images).
3. See what needs to be updated in `docker-compose.yml` to make it work properly with RQ.

Reply:
OK, there are a few changes to be done:

0. Docker image change, obviously
1. Converting beat to rq-scheduler stuff
2. Adding a general worker (to handle emails, periodics etc)
3. Change commas to spaces in queue list

Would the appropriate place to list these manual steps be the [upgrade guide](https://redash.io/help/open-source/admin-guide/how-to-upgrade)? Also, at what point should we modify `getredash/setup`'s [docker-compose.yml](https://github.com/getredash/setup/blob/master/data/docker-compose.yml)?

Reply:
What does converting beat to rq-scheduler means? 

> Change commas to spaces in queue list

Maybe we should support both?

> Would the appropriate place to list these manual steps be the upgrade guide?

I think that we need to add a callout there that will link to the instructions in the CHANGELOG or forum.

> Also, at what point should we modify getredash/setup's docker-compose.yml?

When there is a stable release of v9.

Reply:
> What does converting beat to rq-scheduler means?

Just a simple change in `docker-compose.yml`.

> Maybe we should support both?

RQ natively works with space-separated values, but backwards compatibility makes sense. I'll handle it.

> I think that we need to add a callout there that will link to the instructions in the CHANGELOG or forum.

👍

Reply:
> RQ natively works with space-separated values, but backwards compatibility makes sense. I'll handle it.
> 
> >

It's not mandatory because the list need to be updated anyway, but if it's simple might be worth it because the space based version is confusing.

Reply:
> It's not mandatory because the list need to be updated anyway, but if it's simple might be worth it because the space based version is confusing.

1. The list doesn't need to be updated - I'm suggesting to create a general-purpose worker for things that are _not_ `queries`, `scheduled_queries` or `schemas`.

Comma-based support is available at https://github.com/getredash/redash/pull/4937.

2. Regarding the upgrade info - how's 4685120?